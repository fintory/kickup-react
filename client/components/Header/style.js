import { StyleSheet } from 'aphrodite';

export default StyleSheet.create({
  main: {
    marginTop: 53,
    marginBottom: 50,
  },
  logo: {
    margin: '0 auto',
    width: 51,
    height: 45,
    backgroundImage: 'url(../../assets/logo.png)',
    backgroundRepeat: 'no-repeat',
    backgroundPosition: 'center',
    backgroundSize: '51px 45px',
  },
});
