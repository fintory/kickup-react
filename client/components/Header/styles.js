// @flow
import styled from 'styled-components'

export const Wrapper = styled.header`
  margin-top: 53px;
  margin-bottom: 50px;
`

export const Logo = styled.div`
  margin: 0 auto;
  width: 51px;
  height: 45px;
  background: url(${require('assets/logo.png')}) no-repeat center;
  background-size: 51px 45px;
`
