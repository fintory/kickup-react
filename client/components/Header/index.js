// @flow
import React from 'react'
import { Wrapper, Logo } from './styles'

export default function Header(): React$Element<*> {
  return (
    <Wrapper>
      <Logo />
    </Wrapper>
  )
}
