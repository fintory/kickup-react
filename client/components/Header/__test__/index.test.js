import React from 'react'
import { mount } from 'enzyme'

import Header from '..'

describe('<Header />', () => {
  it('should have an logo in a `div`', () => {
    const wrapper = mount(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})
