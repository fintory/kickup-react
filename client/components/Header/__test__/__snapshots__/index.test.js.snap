// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should have an logo in a \`div\` 1`] = `
.c0 {
  margin-top: 53px;
  margin-bottom: 50px;
}

.c1 {
  margin: 0 auto;
  width: 51px;
  height: 45px;
  background: url(sample-string) no-repeat center;
  background-size: 51px 45px;
}

<Header>
  <styled.header>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "c0",
            "rules": Array [
              "
  margin-top: 53px;
  margin-bottom: 50px;
",
            ],
          },
          "displayName": "styled.header",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "header",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <header
        className="c0"
      >
        <styled.div>
          <StyledComponent
            forwardedComponent={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-bwzfXH",
                  "isStatic": true,
                  "lastClassName": "c1",
                  "rules": Array [
                    "
  margin: 0 auto;
  width: 51px;
  height: 45px;
  background: url(",
                    "sample-string",
                    ") no-repeat center;
  background-size: 51px 45px;
",
                  ],
                },
                "displayName": "styled.div",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-bwzfXH",
                "target": "div",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              }
            }
            forwardedRef={null}
          >
            <div
              className="c1"
            />
          </StyledComponent>
        </styled.div>
      </header>
    </StyledComponent>
  </styled.header>
</Header>
`;
